### ISSUE TITLE
01-Script Paste issue: Line 526

### ISSUE THREAD

**Issue by mattheww95** [BiodataAnalysisGroup/lineagespot#3]
Hello in line 526 of script 01, I have had errors in the path creation.

 It appeared to be concatenating the vcf-file twice, separated by -lineagesport.tsv. I forgot to copy and paste the error unfortunately... 

The line in the script is:  paste(stringr::str_replace(vcf.path, ".vcf", "-lineagespot.tsv"), sep = "")

I have put in my own fix of changing the line to: paste(vcf.path, "_lineagespot.tsv", sep=""). But I do not think this fix is as pretty as the intended outcome is supposed to be :(

I am running the script on version 18.04 of Ubuntu. I have also been specifying the path relative to my home directory (e.g. ~/lineagespot/...)

Thank you for your hard work in the development of this program!

**Comment by nikopech** [comment]
Hi @mattheww95,

Thank you for the comment. 

Unfortunately, I wasn't able to replicate the error you described above. The line you are referring to, `paste(stringr::str_replace(vcf.path, ".vcf", "-lineagespot.tsv"), sep = "")` constructs the path of the output tab-delimited file (which is at the same location as your VCF file). I can't see the way it concatenates the vcf-file twice. 

It would be very useful if you could provide us with the error message, or, if this is not possible, with more information regarding the error (i.e. does your VCF file contains any specific formating?) 

In any case, thank you for your time and effort! Please share your comments and your feedback.

**Comment by mattheww95** [comment]
Error in file(file, ifelse(append, "a", "w")) : 
  cannot open the connection
In addition: Warning message:
In file(file, ifelse(append, "a", "w")) :
  cannot open file '~/lineagespot-lineagespot.tsv-files/Sewage-L2_S10_L001_freebayes.ann.vcf': No such file or directory


Thanks for your response! 

I have installed and run the program on a friends computer and attached the error above! His computer is similarly running Ubuntu version 18.04 as well

n.b. the absolute path was included in the error message, but I have truncated it to include the '~' only for the sake of his privacy.
